projects/canopy/src/lib/card/card.component.scss

```diff
@@ -18,16 +18,22 @@
   &--navigation {
     padding-top: 0;
     transition: box-shadow 250ms;
-    box-shadow: 0 0 0 0 rgba(0, 0, 0, 0.01), 0 0.0625rem 0.0625rem 0 rgba(0, 0, 0, 0.02),
-      0 0.0625rem 0.125rem 0 rgba(0, 0, 0, 0.02), 0 0.125rem 0.25rem 0 rgba(0, 0, 0, 0.02),
-      0 0.1875rem 0.4375rem 0 rgba(0, 0, 0, 0.03), 0 0.5rem 1rem 0 rgba(0, 0, 0, 0.04);
+    box-shadow:
+      0 0 0 0 rgba(0, 0, 0, 0.01),
```

Comment:
These values seem really specific - are they re-used anywhere / designed to match anything else? Just wondering if variables might be useful here

Reply:
That's a good question but don't think it's part of this PR - it only shows as changed because it reformatted it.

I will create a ticket to investigate this though as the `lg-shadow` directive we have in the codebase is slightly different and I wonder if they should be the same 🤔

Reply:
OK cool, thanks 👍